telemetry/telemetry.proto

```diff
@@ -4,6 +4,57 @@ import 'nanopb.proto';
 
 option java_outer_classname="TelemetryProto";
 
-message Telemetry {
-  optional int32 timestamp_ms = 1;
+//Should be sent only after device resets
+message NameIDMessage { //Send the device ID with its corresponding name
+	optional uint32 id = 1;
+	optional string name = 2 [(nanopb).max_size = 32];
+}
+
+message StatisticalAggregate {
+  optional uint32 numSamples = 1;
+  optional uint32 min = 2;
+  optional uint32 max = 3;
+  optional uint32 mean = 4;
+  optional uint32 stdev = 5;
+}
+
+message GeneralReading { //this might want a more specific name?
+	enum READING_TYPE {
+		VOLTAGE = 1;
+		CURRENT = 2;
+		TEMPERATURE = 3;
+	}
+	optional READING_TYPE type = 1;
```

Comment:
Can we get the sensor information (sensor limits, etc.) by querying the sensors in real time, or would we need to store that information in the telemetry board for the NameIDMessage to work? Having to store all the information for each sensor would probably use up a fair chunk of memory, since right now I'm counting 11 fields per device (which will probably be cut down/expanded upon in later revisions, but still).